src/common/low_precision_transformations/include/low_precision/rt_info/precisions_attribute.hpp

```diff
@@ -31,8 +31,7 @@ using PrecisionsAttributePtr = std::shared_ptr<PrecisionsAttribute>;
 
 class LP_TRANSFORMATIONS_API PrecisionsAttribute : public SharedValueAttribute<PrecisionsSharedValue> {
 public:
-    static std::vector<ngraph::element::Type> defaultPrecisions;
-    PrecisionsAttribute(const std::vector<ngraph::element::Type>& precisions = defaultPrecisions);
+    PrecisionsAttribute(const std::vector<ngraph::element::Type>& precisions = { ngraph::element::u8,  ngraph::element::i8 });
```

Comment:
Please, remove default and pass from `LayerTransformation::getDefaultPrecisions()`

Reply:
Fix it, thank you.